java/core/src/java/org/apache/orc/GeospatialColumnStatistics.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
```

Comment:
Please follow the common license header style in the other files.

https://github.com/apache/orc/blob/b6841129e0bc81c86276a7d6f19b497b20dc3bd3/java/core/src/java/org/apache/orc/BinaryColumnStatistics.java#L9-L11